public/Lab-Files/Lesson03/Challenges/wild-animal.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Wild Animal</title>
+</head>
+<body>
+    <div class="main">
+    <h1>The coolest creature</h1>
+        <img src="http://images.fineartamerica.com/images-medium-large-5/portrait-of-a-young-wild-tortoise-tracey-harrington-simpson.jpg"></img>
+
+    <p>The <em>Tortoise</em> is awesome because <strong>They Chill</strong>.</p>
+    
+       <a href="../../../index.html">BACK</a>
+    </div>
+</body>
+
+
+<style>
+    
+    .main {background-color:#F1E4D3;margin:0 auto;text-align: center;}
+    img {border-radius: 50%;margin: 0 auto;display:block;]
+</style>
```

Comment:
The ```<style>``` tags usually go in the ```<head>``` of your document. We put the ```<script>``` at the bottom sometimes for performance, but the styling should be at the top.

Good job though, it seems that you are managing to accomplish what you want. The only thing you need to work on is cleaning up your code a bit, but you are getting the bigger concepts. Great job, keep it up!